audio_service_web/pubspec.yaml

```diff
@@ -9,8 +9,8 @@ environment:
 
```

Comment:
Since `minor` should be in an always-releasable state, I'll need a new process for making platform interface changes. WIP is #912 .